pkg/flink/internal/controller/input_controller.go

```diff
@@ -165,6 +166,20 @@ func (c *InputController) Prompt() (prompt.IPrompt, error) {
 			ASCIICode: []byte{0x1b, 0x7F},
 			Fn:        prompt.DeleteWord,
 		}),
+		// Windows shortcuts
+		prompt.OptionAddKeyBind(prompt.KeyBind{
+			Key: prompt.ControlLeft,
```

Comment:
I can also use Ctrl+Left on my Mac, so should we gate this code with `if runtime.GOOS == "windows"`?

Reply:
Yeah makes sense, updated it